utils/evaluate/evaluate.go

```diff
@@ -56,6 +57,41 @@ func EvaluateResult(result any, metric v1alpha1.Metric, logCtx logrus.Entry) (v1
 	return v1alpha1.AnalysisPhaseSuccessful, nil
 }
 
+func EvalTime(expression string) (time.Time, error) {
+	var err error
+
+	env := map[string]any{
+		"asInt":   asInt,
+		"asFloat": asFloat,
```

Comment:
Let's remove asInt, asFloat and just use the built in funcs https://expr-lang.org/docs/language-definition#int

Reply:
fixed!